packages/core/src/layout/Header/Header.test.tsx

```diff
@@ -26,33 +26,33 @@ jest.mock('react-helmet', () => {
 });
 
 describe('<Header/>', () => {
-  it('should render with title', () => {
+  it('should render with title', async () => {
```

Comment:
What is the reason for these to become `async`?

Reply:
Ops! I added async code for some purpose and then removed it but forgot to delete that keywords